app/common/src/Device.js

```diff
@@ -3,9 +3,16 @@
 import { Dimensions } from 'react-native';
 
 function renewDimensions() {
-  return Dimensions.get('screen');
+  try {
+    return Dimensions.get('screen');
+  } catch (e) {
+    return { width: 0, height: 0 };
+  }
 }
 
+// Tablets have often dimensions ratio 3:2-4:3, phones have often ratio 16:9
+const WIDE_DEVICE_THRESHOLD = 0.625; // 16:10 is max acceptable for screen rotation
```

Comment:
What devices did you try with this magic constant? I tried iPad 10.5 inch and it doesn't look good (and also far away from the design again).

Reply:
It's explained in comments. It's dimensions ratio that "empirically" make sense to display search results and map on one screen.
What else doesn't look good? Pls next time make a screenshot. It looks good to me.

Now I see map and results are actually connected. Well it's not optimization at all, but pretty much a feature.
We can considered this PR as a one step to final design...

Reply:
My first confusion was: _"Why is he talking about threshold 0.625 if 16:10 is clearly 1.6... Aha, it's actually 10:16 - that is confusing."_

Also, it doesn't look OK for me: 
![simulator screen shot - ipad pro 10 5-inch - 2018-01-23 at 15 23 02](https://user-images.githubusercontent.com/978368/35265048-91f51bb8-0051-11e8-8f47-9471580d37cc.png) and since you are using fixed width the landscape is also far away from what is designed here: https://zpl.io/aML67Y7 (search is about 45% of the screen and not just 360 points). It looks weird basically only because of the "360".